### ISSUE TITLE
Error on training "TypeError: expected str, bytes or os.PathLike object, not NoneType"

### ISSUE THREAD

**Issue by devstermarts** [caillonantoine/msprior#2]
Hi @caillonantoine, i'm running into the following error while training on Colab with miniconda (https://repo.anaconda.com/miniconda/Miniconda3-py39_4.12.0-Linux-x86_64.sh) installed : 

```bash
!/content/miniconda/bin/msprior train --db_path /content/drive/MyDrive/msprior_output --name trainingname
```

```bash
I0605 09:41:40.950367 140406031978880 train.py:52] parsing configuration
I0605 09:41:40.950587 140406031978880 resource_reader.py:50] system_path_file_exists:msprior/configs/decoder_only.gin
Traceback (most recent call last):
  File "/content/miniconda/bin/msprior", line 8, in <module>
    sys.exit(main())
  File "/content/miniconda/lib/python3.9/site-packages/msprior_scripts/main_cli.py", line 28, in main
    app.run(module.main)
  File "/content/miniconda/lib/python3.9/site-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/content/miniconda/lib/python3.9/site-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
  File "/content/miniconda/lib/python3.9/site-packages/msprior_scripts/train.py", line 59, in main
    gin.parse_config_files_and_bindings(
  File "/content/miniconda/lib/python3.9/site-packages/gin/config.py", line 2497, in parse_config_files_and_bindings
    includes_and_imports = parse_config_file(config_file, skip_unknown)
  File "/content/miniconda/lib/python3.9/site-packages/gin/config.py", line 2448, in parse_config_file
    if existence_check(config_file_with_prefix):
  File "/content/miniconda/lib/python3.9/site-packages/gin/resource_reader.py", line 52, in system_path_file_exists
    path = _parse_config_path(config_path)
  File "/content/miniconda/lib/python3.9/site-packages/gin/resource_reader.py", line 89, in _parse_config_path
    path = os.path.join(os.path.dirname(file_sys_path), filename)
  File "/content/miniconda/lib/python3.9/posixpath.py", line 152, in dirname
    p = os.fspath(p)
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

Pytorch version is: pytorch-lightning-1.9.0

**Comment by caillonantoine** [comment]
Fixed in 12d5afa !

**Comment by caillonantoine** [comment]
Upgrade your msprior install with `pip install -U acids-msprior` once [this action](https://github.com/caillonantoine/msprior/actions/runs/5200183697) is over
